Resources/views/Action/obtain_checkout_token.html.twig

```diff
@@ -5,8 +5,16 @@
 
     <form method="post" class="formular" id="paymentFrameForm">
         <iframe id="paymentIframe" src="{{ frame_src }}" style="border:none; height:200px;"></iframe>
-        <br/>
-        <button class="btn btn-success" type="submit">{{ 'coreshop.ui.buy'|trans }}</button>
+        <div class="container">
+            <div class="row bg-classes-transparent">
+                <div class="12 col-sm-6 text-center text-sm-left">
+                    <button class="btn btn-success" type="submit">{{ 'coreshop.ui.buy'|trans }}</button>
+                </div>
+                <div class="12 col-sm-6 text-center text-sm-right yorxs-button-link ">
+                    <a class="btn btn-secondary" href="{{ coreshop_path(order, 'coreshop_order_revise', {'token': order.token}) }}">{{ 'coreshop.ui.back'|trans }}</a>
```

Comment:
not sure if it is enough, normally the order should get canceled first. otherwise the order stays in state "new"

Reply:
Just checked. It's enough and is the same behaviour like it is if i cancel PayPal Payment.
Order ist in State "NEW - awaiting payment" and customer is redirected to the /order/revise to select another payment method.

If the customer choose on this page "cancel order" the cart will be recreated and the order is set to "Canceld".

Reply:
that is true. all fine then